facefusion/config.py

```diff
@@ -1,24 +1,34 @@
 from configparser import ConfigParser
 from typing import Any, Optional, List
-
+import os.path 
 from facefusion.filesystem import resolve_relative_path
+import facefusion.globals
 
 CONFIG = None
 
 
-def get_config() -> ConfigParser:
-	global CONFIG
-
-	if CONFIG is None:
-		config_path = resolve_relative_path('../facefusion.ini')
-		CONFIG = ConfigParser()
-		CONFIG.read(config_path, encoding = 'utf-8')
-	return CONFIG
-
+def get_config(config_file_path: str = None) -> ConfigParser:
+    global CONFIG
+	
+    if CONFIG is None:  # Only read from file if CONFIG is not initialized
```

Comment:
1. we do not use comments in our code - if they are needed, the code is considered bad or hard to understand
2. there is no need for all the conditions - you just make sure that the `facefusion.globals.ini_path` is set via `apply_args()`